express/code/blocks/prompt-marquee/prompt-marquee.css

```diff
@@ -0,0 +1,351 @@
+.prompt-marquee {
+  --prompt-marquee-desktop-padding-block: 80px;
+  --prompt-marquee-desktop-media-max-width: 571px;
+  --prompt-marquee-media-aspect-ratio: 16 / 9;
+  padding-left: var(--spacing-300);
+  padding-right: var(--spacing-300);
+  padding-top: var(--spacing-600);
+  margin: auto;
+  max-width: 900px;
+  padding-bottom: var(--spacing-600);
+}
+
+.prompt-marquee .icon.express-logo {
+  width: fit-content;
+  margin-left: 0px;
+  padding: 0px;
+}
+
+.prompt-marquee > div {
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacing-500);
+  padding: 0;
+}
+
+.prompt-marquee .column {
+  padding-left: 0;
+  padding-right: 0;
+  text-align: left;
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+  gap: var(--spacing-100);
+}
+
+.prompt-marquee .column p {
+  max-width: 500px;
+  margin: 0;
+}
+
+/* Mobile font sizing adjustments */
+.prompt-marquee h1,
+.prompt-marquee h2,
+.prompt-marquee h3,
+.prompt-marquee h4,
+.prompt-marquee h5,
+.prompt-marquee h6 {
+  margin: 0;
+  color: var(--color-gray-900);
+  font-style: normal;
+  font-weight: 900;
+  /* Mobile-first sizes */
+  font-size: var(--heading-font-size-m);
+  line-height: var(--heading-line-height);
+}
+
+/* Mobile-first paragraph sizing */
+.prompt-marquee p { font-size: var(--body-font-size-l); font-weight: 400; line-height: 130%; }
+
+/* Upscale headings on larger viewports */
+@media (min-width: 768px) {
+  .prompt-marquee h1,
+  .prompt-marquee h2,
+  .prompt-marquee h3,
+  .prompt-marquee h4,
+  .prompt-marquee h5,
+  .prompt-marquee h6 {
+    font-size: var(--Global-Typography-Size-Headings-Heading-2XL, 32px);
+    line-height: var(--Global-Typography-Line-height-Headings-Heading-2XL, 36px);
+  }
+}
+
+.prompt-marquee .button-container { display: block; }
+
+.prompt-marquee .cta-with-input {
+  display: flex;
+  gap: var(--spacing-300);
+  align-items: stretch;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 12px 24px;
+  border: none;
+}
+
+.prompt-marquee .cta-with-input .prompt-marquee-input {
+  font-family: var(--body-font-family, 'Adobe Clean', 'Helvetica Neue', Helvetica, Arial, sans-serif);
+  font-size: var(--body-font-size-m);
+  line-height: 1.4;
+}
+
+main .section .prompt-marquee .button-container{
+  padding: 0;
+  margin-top: var(--spacing-300);
+}
+
+main .prompt-marquee .cta-with-input .prompt-marquee-input { order: 0; width: 100%; }
+main .prompt-marquee .cta-with-input a.button:any-link,
+main .prompt-marquee .cta-with-input .con-button { 
+  order: 1;
+  white-space: nowrap;
+  box-sizing: border-box;
+  margin: 0;
+  border: none;
+  padding: 12px 24px;
+  min-width: auto; /* shrink to text width + padding */
+  display: inline-flex; /* reduce extra inline-box space */
+  align-items: center;
+  justify-content: center;
+  line-height: 1.2; /* tighter vertical metrics around text */
```

Comment:
There's a _lot_ of comments like this that are a bit bloated. Please manually reduce to what is useful